build-plugin/src/main/kotlin/ThunderbirdPlugins.kt

```diff
@@ -0,0 +1,6 @@
+object ThunderbirdPlugins {
```

Comment:
Is this used anywhere?

Reply:
Not now as in groovy build files there is no access to it. Once kts is there it could be used.

Reply:
I planed to use it for one of the next pull-requests comming up